Error using EX-106

I'm using a EX-106 motor with firmware 23 on a Ubuntu 14.04/Indigo machine and try to use it with dynamixel_motor. When starting up, the following it printed to console:
```
setting /run_id to f45886d2-7114-11e6-9a6b-00904c0df43e
process[rosout-1]: started with pid [9976]
started core service [/rosout]
process[pan_controller_spawner-2]: started with pid [9993]
process[dynamixel_manager-3]: started with pid [9994]
[INFO] [WallTime: 1472824550.573297] your_dynamixel_usb_port controller_spawner: waiting for controller_manager dxl_manager to startup in global namespace...
/opt/ros/indigo/lib/python2.7/dist-packages/dynamixel_driver/dynamixel_serial_proxy.py:92: SyntaxWarning: The publisher should be created with an explicit keyword argument 'queue_size'. Please see http://wiki.ros.org/rospy/Overview/Publishers%20and%20Subscribers for more information.
  self.motor_states_pub = rospy.Publisher('motor_states/%s' % self.port_namespace, MotorStateList)
/opt/ros/indigo/lib/python2.7/dist-packages/dynamixel_driver/dynamixel_serial_proxy.py:93: SyntaxWarning: The publisher should be created with an explicit keyword argument 'queue_size'. Please see http://wiki.ros.org/rospy/Overview/Publishers%20and%20Subscribers for more information.
  self.diagnostics_pub = rospy.Publisher('/diagnostics', DiagnosticArray)
[INFO] [WallTime: 1472824550.784866] pan_tilt_port: Pinging motor IDs 1 through 25...
[ERROR] [WallTime: 1472824552.816269] Exception thrown while getting attributes for motor 1 - 106
[ERROR] [WallTime: 1472824552.836134] Exception thrown while getting attributes for motor 1 - 106
[ERROR] [WallTime: 1472824552.855784] Exception thrown while getting attributes for motor 1 - 106
[ERROR] [WallTime: 1472824552.875768] Exception thrown while getting attributes for motor 1 - 106
[ERROR] [WallTime: 1472824552.897048] Exception thrown while getting attributes for motor 1 - 106
[INFO] [WallTime: 1472824552.899264] pan_tilt_port: Found 0 motors , initialization complete.
```
The 106 error code suggests the servo type was read correctly, but apparently, something goes wrong. Any ideas?

--- a/dynamixel_driver/src/dynamixel_driver/dynamixel_const.py
+++ b/dynamixel_driver/src/dynamixel_driver/dynamixel_const.py
@@ -235,7 +235,15 @@ DXL_MODEL_TO_PARAMS = \
            'rpm_per_tick':       0.111,
            'features':           []
          },
-    107: { 'name':               'EX-106',
+    106: { 'name':               'EX-106',
+           'encoder_resolution': 4096,
+           'range_degrees':      250.92,
+           'torque_per_volt':    10.9 / 18.5,                      # 10.9 NM @ 18.5V
+           'velocity_per_volt':  (91 * RPM_TO_RADSEC) / 18.5,      #  91 RPM @ 18.5V
+           'rpm_per_tick':       0.111,
+           'features':           [DXL_SENSED_CURRENT_L]
+         },     
+    107: { 'name':               'EX-106+',
            'encoder_resolution': 4096,
            'range_degrees':      250.92,
            'torque_per_volt':    10.9 / 18.5,                      # 10.9 NM @ 18.5V

